### PR TITLE
Update mlir-aie and mlir-air versions; enable support for conv2d with bigger image sizes

### PR DESCRIPTION
**Pull request by erwei-xilinx**

### CODE REVIEW

**Comment by yzhang93** [comment]
Thanks! I think you'll also need to update mlir-aie wheel. https://github.com/nod-ai/iree-amd-aie/blob/5a7ebc899a0aa26706ee2e52f85bda5609948bb7/.github/workflows/ci.yml#L114

File: runtime/src/iree-amd-aie/aie_runtime/test/DeviceModelTest.cpp [hunk 1]
```diff
@@ -413,8 +413,8 @@ TEST_P(AMDAIENPUDeviceModelParameterizedSixTupleNPU4ColTestFixture,
     auto destWireb = STRM_SW_PORT_TYPE_TO_WIRE_BUNDLE(destSw);
     auto deviceModelIsLegal = deviceModel.isLegalMemtileConnection(
         c, r, srcSw, srcChan, destSw, dstChan);
-    auto targetModelIsLegal = targetModel.isLegalMemtileConnection(
-        srcWireB, srcChan, destWireb, dstChan);
+    auto targetModelIsLegal = targetModel.isLegalTileConnection(
+        c, r, srcWireB, srcChan, destWireb, dstChan);
```
Comment (makslevental) [inline runtime/src/iree-amd-aie/aie_runtime/test/DeviceModelTest.cpp:417]: Ah this isn't gonna work - we're not using Air target model we're using [Device model](https://github.com/nod-ai/iree-amd-aie/blob/5a7ebc899a0aa26706ee2e52f85bda5609948bb7/runtime/src/iree-amd-aie/aie_runtime/iree_aie_runtime.h#L227). Do your changes depend on the recent updates to AIETargetModel?

Comment (erwei-xilinx) [reply runtime/src/iree-amd-aie/aie_runtime/test/DeviceModelTest.cpp:417]: Yeah this recent PR is introducing changes to AIETargetModel: https://github.com/erwei-xilinx/mlir-aie/commit/c04ffea6dc2430b65557feca3ff6a7f55a2e64b5

**Comment by erwei-xilinx** [comment]
> Thanks! I think you'll also need to update mlir-aie wheel.
> 
> https://github.com/nod-ai/iree-amd-aie/blob/5a7ebc899a0aa26706ee2e52f85bda5609948bb7/.github/workflows/ci.yml#L114

Ah good point. Thanks for the tip!

**Comment by erwei-xilinx** [comment]
Thanks for the pointers and fixes, @makslevental! All `EXPECT_NE` for memtile connectivity are now gone.

**Review by yzhang93 — APPROVED** [review]

**Comment by erwei-xilinx** [comment]
> Thanks! I think you'll also need to update mlir-aie wheel.
> 
> https://github.com/nod-ai/iree-amd-aie/blob/5a7ebc899a0aa26706ee2e52f85bda5609948bb7/.github/workflows/ci.yml#L114

Mlir-aie wheel updated.

**Comment by erwei-xilinx** [comment]
Updating the mlir-aie [wheel ](https://github.com/Xilinx/mlir-aie/commit/1acb6fe98e1613bc5e1b6bb8cf02cbf555d2fd3a) beyond this commit seems to be causing the CI tests to fail. This PR shall roll back the wheel version https://github.com/nod-ai/iree-amd-aie/pull/557.

**Pull request merged**